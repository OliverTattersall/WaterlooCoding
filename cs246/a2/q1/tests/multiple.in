r a 101/
r b 110/
pa,
pb,
da
db
+a101/
pa,
pb,
da
db
q
