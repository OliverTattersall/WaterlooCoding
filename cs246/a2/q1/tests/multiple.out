1,0,1

1,1,0

5
6
1,0,1,1,0,1

1,1,0

45
6
